### PR TITLE
 CTPPS: standard descriptions filled (9_2_X)

### PR DESCRIPTION
**Pull request by forthommel**
backport of #19829

This PR populates the `fillDescriptions` methods for all `RecoCTPPS/TotemRPLocal` plugins to allow the automatic generation of the associated `cfi` files.
Work done prior to the foreseen HLT studies for CTPPS.

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
#19829 is almost ready to be signed by reco (just waiting for the results of the jenkins tests).
If you want that also this backport get signed, please update with the latest adjustments integrated in the master (latest four commits to be cherry picked)

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
This commit
https://github.com/cms-sw/cmssw/pull/19831/commits/e68bf5c46606c97d43810bc570a09f869905d883
is different from the corresponding one in the master
https://github.com/cms-sw/cmssw/pull/19829/commits/7ab86bf16ef419cdb21c3adcc4365e1e9a092fa6
as it misses the fix to recoCTPPS_cff.py

How can it be? Did you cherry pick the one in the master, or what?

**Comment by forthommel** [comment]
Indeed, that is strange... I might have messed up this file while cherry-picking one of these last 4 commits. It seems fine now!

File: RecoCTPPS/Configuration/python/recoCTPPS_cff.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 from RecoCTPPS.TotemRPLocal.totemRPLocalReconstruction_cff import *
 from RecoCTPPS.TotemRPLocal.ctppsDiamondLocalReconstruction_cff import *
-from RecoCTPPS.TotemRPLocal.ctppsLocalTrackLiteProducer_cfi import ctppsLocalTrackLiteProducer
+from RecoCTPPS.TotemRPLocal.ctppsLocalTrackLiteProducer_cff import ctppsLocalTrackLiteProducer
```
Comment (perrotta) [inline RecoCTPPS/Configuration/python/recoCTPPS_cff.py]: ctppsPixelLocalReconstruction is missing in this recoCTPPS sequence

The reason is that #19909. is not yet merged in 92X: this PR cannot be merged befere that one.

**Comment by perrotta** [comment]
This PR depends on #19909: it cannot be merged befere that one.

**Comment by perrotta** [comment]
+1
- Automatically generated configurations are identical to the previous explicit ones
- No difference in the results of the jenkins tests, as expected
- As a further check, some 200 evts from the wf 133.78 were also run with and without this PR, and again there were no differences on the reco outputs

**Comment by perrotta** [comment]
(I removed my previous comment and signature because I added it to the wrong PR...)

**Comment by perrotta** [comment]
+1
- Even though it was merged in the master in the different order, this PR can be also merged before #19909. Let see if that different order will require some rebasing and adjustment of #19909: in any case, merging this one...
- Automatically generated configurations are identical to the previous explicit ones: thus no change is expected in outputs
- No difference in the results of the jenkins tests, as expected

**Comment by davidlange6** [comment]
so in the end i merged #19909 first - so this needs a rebase.

**Comment by perrotta** [comment]
:-)

Please @forthommel rebase to the top 9_2_X (or just wait next IB and rebase to it)

**Comment by perrotta** [comment]
Still RecoCTPPS/Configuration/python/recoCTPPS_cff.py need to be fixed (cfi -> cff)

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
- Automatically generated configurations are identical to the previous explicit ones: thus no change is expected in output
- The differences observed in the results of the jenkins test are due to the other commits tested together

**Comment by davidlange6** [comment]
merge

**Pull request merged**